addons/delivery/models/delivery_carrier.py

```diff
@@ -279,8 +294,6 @@ def _get_default_custom_package_code(self):
     # Fixed price shipping, aka a very simple provider #
     # ------------------------------------------------ #
 
-    fixed_price = fields.Float(compute='_compute_fixed_price', inverse='_set_product_fixed_price', store=True, string='Fixed Price')
-
```

Comment:
avoid moving fields/methods around for now (except the ones where you remove the file ofc)